src/transcript.rs

```diff
@@ -16,18 +15,20 @@ pub trait TranscriptProtocol {
 
 impl TranscriptProtocol for Transcript {
     fn append_commitment(&mut self, label: &'static [u8], comm: &Commitment<Bls12_381>) {
-        self.append_message(label, &to_bytes![comm].unwrap());
```

Comment:
i cant find to_bytes macro anymore, i replaced it by serialize but unsure if its correct